### DISCUSSION TITLE
Shareable components

### DISCUSSION THREAD

**Discussion by Quin452** [uxbox/uxbox#493] [General]
I am very excited for this project; from what I've seen so far, it looks to be a great competitor in the near future.
To be honest, I wish I could help out on the dev side, but I haven't a clue where to start!

However, I was wondering, would components be shareable between projects, and may even across the community?

Say, for instance, some hand-drawn wireframe components, so people can quickly prototype?

**Comment by myfunnyandy** [comment]
Hi @Quin452, great to hear about your enthusiasm for the project :)

The alpha release that we will launch next week will have components. Components are reusable elements (single objects or a group of them) shareable between files and projects of a team. Besides, is in our plan to give libraries (that can contain components and other types of assets) the ability to go public, enabling them to be shared throughout the community. This will be in the future as we still need to shape this feature that will require so much consideration.

About contributing, soon we will release documentation to help people participate in the project. If you are interested stay tuned because we really want to involve the community in the development of Penpot.
